### ISSUE TITLE
Use mnist validation or test set when querying the served model

### ISSUE THREAD

**Issue by mabu-github** [intro-to-ml-with-kubeflow/intro-to-ml-with-kubeflow-examples#81]
When evaluating the model, the validation or test set should be used because classifiers may overfit and yield perfect results on the training set.

I would suggest changing
`batch_xs, batch_ys = mnist.train.next_batch(1)`
to
`batch_xs, batch_ys = mnist.test.next_batch(1)`
in the section about querying the model.
